docs/supplychain-sbom.md

```diff
@@ -448,27 +457,27 @@ Not to be confused with the [SBOM Author](#sbom-author--role-) role.
-| 🟨 | Licenses                       | Yes      | SPDX License | bom.components[].licenses[]                                           |      |                                            |
-| 🟥 | Public Code Repository         | Yes      |              | bom.metadata.component.externalReferences[].vcs                       |      |                                            |
+| 🟥 | Unique ID, Product ID          | Yes      | PURL         | bom.components[].purl                                                 | packages[].externalRefs.referenceCategory = "PACKAGE-MANAGER", packages[].externalRefs.referenceType = "purl", packages[].externalRefs.referenceLocator | CRA-AII(3), NTIA-SBOM, CRA-AV              |
+| 🟥 | Purpose, Intended Use          | Yes      | Text         | bom.components[].description                                          | packages[].comment | CRA-AII(4)                                 |
+| 🟨 | Licenses                       | Yes      | SPDX License | bom.components[].licenses[]                                           | packages[].licenseConcluded, packages[].licenseConcluded |                                            |
+| 🟥 | Public Code Repository         | Yes      |              | bom.metadata.component.externalReferences[].vcs                       | packages[].externalRefs.referenceCategory = "PERSISTENT_ID", packages[].externalRefs.referenceType = "gitoid", packages[].externalRefs.referenceLocator |                                            |
 | 🟥 | Intended for Commercial Use    | No       | Boolean      |                                                                       |      | CRA-Rec-15                                 |
 | 🟥 | Open-Source Software Steward   | No       | URL          |                                                                       |      | CRA                                        |
 | 🟥 | Code Commit Revision           | No       |              |                                                                       |      |                                            |
-| 🟨 | Code Repository                | Yes      |              | bom.components[].externalReferences[].vcs                             |      |                                            |
-| 🟨 | Owner Name (Author)            | Yes      | Text, URL    | bom.components[].supplier                                             |      | CRA-AII(1), NTIA-SBOM, DE-TR.5.2.2, CRA-AV |
+| 🟨 | Code Repository                | Yes      |              | bom.components[].externalReferences[].vcs                             | bom.metadata.component.externalReferences[].vcs                       | packages[].externalRefs.referenceCategory = "PERSISTENT_ID", packages[].externalRefs.referenceType = "gitoid", packages[].externalRefs.referenceLocator |                                            |
```

Comment:
Ah! Seems you found an error on my side. Code Repository needs only to be listed once. :sweat_smile:

Reply:
I totally missed this, I fixed it in my branch